setup/includes/upgrades/common/3.0.0-update-tvs-list-legacy.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Common upgrade script for modify modTemplateVar with 'list-multiple-legacy' input type
+ *
+ * @var modX $modx
+ * @package setup
+ */
+
+use MODX\Revolution\modTemplateVar;
+
+/** @var modTemplateVar $tvs_list_legacy */
+$tvs_list_legacy = $modx->getCollection(modTemplateVar::class, ['type' => 'list-multiple-legacy']);
```

Comment:
Let's use PSR-12 for a new code. 
```suggestion
$legacyMultipleList = $modx->getCollection(modTemplateVar::class, ['type' => 'list-multiple-legacy']);
```